owasp-top10-2017-apps/a5/tictactoe/Makefile

```diff
@@ -0,0 +1,38 @@
+.SILENT:
+.DEFAULT_GOAL := help
+
```

Comment:
I think it would be cool to add a `healthcheck`, so the user can be notified when the app starts successfully. We are doing this in other `Makefiles`, like [this one](https://github.com/globocom/secDevLabs/blob/master/owasp-top10-2017-apps/a4/vinijr-blog/Makefile)

Reply:
Thank you, @Krlier. I added a healthcheck route